### PR TITLE
Pdffulltext

### PR DESCRIPTION
**Pull request by stuartf**
requesting pull of this from oaeproject repo so that travis will upload the logs when it fails

### CODE REVIEW

**Comment by simong** [comment]
From those logs
```
[2015-01-07T18:43:51.511Z] ERROR: oae-preview-processor/10850 on testing-worker-linux-8-1-9724-linux-20-46226010: Could not convert the PDF to plain text (contentId=c:localhost:Qkmbo8nJE, stdout="")
    Error: Command failed: /bin/sh: 1: pdftotext: not found
    
        at ChildProcess.exithandler (child_process.js:648:15)
        at ChildProcess.emit (events.js:98:17)
        at maybeClose (child_process.js:756:16)
        at Process.ChildProcess._handle.onexit (child_process.js:823:5)
    --
    stderr: /bin/sh: 1: pdftotext: not found
```

Presumable it needs the full path towards the binary (we seem to do similar things for pdf2htmlex and pdftk)

**Comment by simong** [comment]
We might need to play with scoring/boosting of documents. The cutoff point is currently set at `0.09` which might still be too high. The has_child query results in generally low document scores (as the content bodies tend to be quite large)

**Comment by mrvisser** [comment]
Discussing search analysis requirements sounds remarkably similar a weather forecast :)

File: node_modules/oae-preview-processor/tests/test-previews.js [hunk 1]
```diff
@@ -525,6 +525,13 @@ describe('Preview processor', function() {
                                 assert.ok(_.find(previews.files, function(file) { return file.filename === 'page.1.html'; }));
                                 assert.ok(!_.find(previews.files, function(file) { return file.filename === 'page.2.html'; }));
 
+                                // The PDF has 1 page, there should only be 1 corresponding txt file
```
Comment (nicolaasmatthijs) [inline node_modules/oae-preview-processor/tests/test-previews.js]: Shouldn't we have a test for a multi-page PDF where we verify that multiple `page.x.txt` files?

**Pull request merged**